### PR TITLE
Add support for Italian keyboard on console

### PR DESCRIPTION
**Pull request by filippocarletti**
* Load the keymap: loadkmap </usr/share/keymaps/it.map.bin
* To build a binary keymap on CentOS (as root): loadkeys -b it.map.gz -u -q >it.map.bin

### CODE REVIEW

File: files/lib/preinit/95_keymap [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Copyright 2022 Nethesis
+
+KMAPDIR="/usr/share/keymaps"
+[ -n "$lang" ] && [ -s $KMAPDIR/"$lang".map.bin ] && cat $KMAPDIR/"$lang".map.bin | loadkmap
```
Comment (gsanchietti) [inline files/lib/preinit/95_keymap:4]: Where is defined the `$lang` variable and what is the default value?

File: files/lib/preinit/95_keymap [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Copyright 2022 Nethesis
```
Comment (gsanchietti) [inline files/lib/preinit/95_keymap:1]: If you want to add an header, please see https://github.com/NethServer/nextsecurity/blob/master/README.md#license-headers

**Review by gsanchietti — CHANGES_REQUESTED** [review]

File: files/lib/preinit/95_keymap (hunk 1 above)
Comment (gsanchietti) [reply files/lib/preinit/95_keymap:4]: Reading the description, I guess the variable comes from grub and the default is empty. Am I right?

Comment (filippocarletti) [reply files/lib/preinit/95_keymap:4]: Right

File: files/lib/preinit/95_keymap (hunk 2 above)
Comment (filippocarletti) [reply files/lib/preinit/95_keymap:1]: Do we need a license in a one-line file? :-)

**Comment by filippocarletti** [comment]
We still have to work on sysupgrade, we can maintain grub options.
AFAIK, no, there's no way to view the keyboard layout.

**Review by gsanchietti — APPROVED** [review]
Let's go for a very basic implementation for now.

**Pull request merged**